src/instruments/src/EFB/ATC/ATC.tsx

```diff
@@ -18,11 +18,16 @@ export const ATC = () => {
     const [currentLatitude] = useSimVar('GPS POSITION LAT', 'Degrees', 5000);
     const [currentLongitude] = useSimVar('GPS POSITION LON', 'Degrees', 5000);
     const [atisSource] = usePersistentProperty('CONFIG_ATIS_SRC', 'FAA');
+    const [lastUpdate, setLastUpdate] = useState<string>();
 
     useEffect(() => {
-        loadAtc();
-        setInterval(() => loadAtc(), 2 * 60 * 1000);
-    }, []);
+        const interval = setInterval(() => {
+            loadAtc();
+            setLastUpdate(new Date().toString());
+        }, 60 * 1000);
+
+        return () => clearInterval(interval);
+    }, [lastUpdate]);
```

Comment:
Why have you added `lastUpdate` as a dependency to this hook? You're not using it.

Reply:
the hook executes everytime lastUpdate change.
I had to use this 'hack' otherwise the "currentLatitude" and "currentLongitude" wasn't updated within the setInterval()